### ISSUE TITLE
W value seems to be polled more often than others

### ISSUE THREAD

**Issue by ph30n1x** [Emilv2/huawei_solar#38]
Hi,

I had to apply an update to HA and to the solar code to get it back up and running, but now it looks like the script is polling certain value more than others. It is for sure appears to be grabbing the W value (Invertor output) every 5 seconds rather than 30 seconds so it is completely destroying the overall data. Other values appear to be only being captured every 30 seconds as previously 

Has anyone else experienced this? I have no idea how to troubleshoot / rectify this (the huawei_solar was manually installed)

I have attached a csv to show how the W.mean_value is returning values more than it should

[2021-10-21-13-58 Chronograf Data.csv](https://github.com/Emilv2/huawei_solar/files/7386368/2021-10-21-13-58.Chronograf.Data.csv)

**Comment by Emilv2** [comment]
I'm not sure what is going on there, all the state and attributes values are update in one function call. There is no way to update one without the other.

**Comment by ph30n1x** [comment]
@Emilv2 any suggestion? Would this require me to delete the custom integration and then re-add it?

**Comment by ph30n1x** [comment]
@Emilv2 I deleted the integration, installed HACS and added the huawei solar via HACS. exact same results!

**Comment by Emilv2** [comment]
Yes, I don't see how deleting it would change anything. I have rewritten a lot of the code of this sensor, does the issue still occur with the latest version?

**Comment by ph30n1x** [comment]
@Emilv2 the issue still occurs and now that i have a battery the value is more important, all i am getting is garbage data
